include/cuco/static_map.cuh

```diff
@@ -82,8 +83,9 @@ struct is_bitwise_comparable<T, std::enable_if_t<std::has_unique_object_represen
  * @brief A GPU-accelerated, unordered, associative container of key-value
  * pairs with unique keys.
  *
- * Allows constant time concurrent inserts or concurrent find operations (not
- * concurrent insert and find) from threads in device code.
+ * Allows constant time concurrent inserts or concurrent find operations from threads in device
+ * code. Concurrent insert and find are supported only if the pair type is packable (see
+ * `cuco::detail::is_packable` constexpr).
```

Comment:
Similar to [`std::atomic<T>::is_always_lockfree`](https://en.cppreference.com/w/cpp/atomic/atomic/is_always_lock_free), we should add a static member that can be queried to indicate if concurrent insert/find is supported. Maybe `has_concurrent_insert_find`?